docs/admin_api/user_admin_api.md

```diff
@@ -197,11 +199,12 @@ The following parameters should be set in the URL:
   - `shadow_banned` - Users are ordered by `shadow_banned` status.
   - `displayname` - Users are ordered alphabetically by `displayname`.
   - `avatar_url` - Users are ordered alphabetically by avatar URL.
+  - `creation_ts` - Users are ordered by when the users was created in s (Unix timestamp).
```

Comment:
Could you make it return a value in milliseconds instead of seconds please? The `users` table is an old one and as such as a few inconsistencies with the rest of Synapse, one of them being using seconds instead of milliseconds. The rest of Synapse uses milliseconds, so I think it'd be nice if we could try to head towards consistency here.

Reply:
Yes I can do that, but then there is a mismatch to [Query user API](https://github.com/matrix-org/synapse/blob/develop/docs/admin_api/user_admin_api.md#query-user-account)
What do we do with the Query user API? Also change to `ms`? With a version increment?

I would like to prefer that both APIs returns the same values.

Reply:
Ah yeah somehow I've missed that the query user API also returns a value in seconds. Ideally I'd like it to change to ms too (with a v3 of the endpoint in order not to confuse users). I'd suggest doing that in a separate PR though.

Reply:
I do not like a `v3` for this small change.
But we can start here to use `ms`.

Reply:
Yes definitely. And agreed that v3 is maybe a bit overkill for such a small change, but I also don't think we should change the unit in the v2 endpoint, lest we break existing scripts using it.